app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -74,23 +81,25 @@
       app:layout_constraintTop_toBottomOf="@+id/error_text"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintEnd_toEndOf="parent"/>
+
     <TextView
       android:id="@+id/forgot_pin"
       android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_marginTop="10dp"
+      android:layout_height="48dp"
+      android:layout_marginTop="12dp"
```

Comment:
This should be 4 dp, because you already have a 8dp padding in pinView.

Reply:
Done.

Reply:
This has not been updated yet.